packages/node_modules/@ciscospark/react-component-people-list/src/index.js

```diff
@@ -16,56 +19,99 @@ const propTypes = {
-  onItemClick: PropTypes.func
+  onItemClick: PropTypes.func,
+  onDisplayUsers: PropTypes.func
 };
 
 const defaultProps = {
-  onItemClick: () => {}
+  onItemClick: () => {},
+  onDisplayUsers: () => {}
 };
 
-
 function PeopleList({
   items,
-  onItemClick
+  onItemClick,
+  onDisplayUsers
 }) {
   if (!items || !items.length) {
     return null;
   }
 
-  function generateGroupList(people) {
-    return people.map((p) => {
-      const handleItemClick = () => {
-        onItemClick(p);
-      };
+  const totalRows = [];
+
+  items.forEach((item) => {
+    // Header for item
+    if (item.label) {
+      totalRows.push({type: 'header', details: item.label});
+    }
+    // Rows for each person
+    item.people.forEach((person) => {
+      totalRows.push({type: 'person', details: person});
+    });
+  });
 
+  // disable react/prop-types because eslint thinks rowRenderer is the main render
+  // eslint-disable-next-line react/prop-types
+  function rowRenderer({key, index, style}) {
+    if (totalRows[index].type === 'header') {
       return (
+        <div style={style} className={classNames('ciscospark-people-group', styles.group)} key={totalRows[index].details}>
+          {
+            totalRows[index].details
+            && <h3 className={classNames('ciscospark-people-group-title', styles.title)}>{totalRows[index].details}</h3>
+          }
+        </div>
+      );
+    }
+
+    const person = totalRows[index].details;
+
+    const handleItemClick = () => {
+      onItemClick(person);
+    };
+
+    return (
+      <div key={key} style={style}>
         <Person
-          displayName={p.displayName || p.name}
-          key={p.id}
+          displayName={person.displayName || person.name}
+          key={person.id}
           onClick={handleItemClick}
-          {...p}
+          emailAddress={person.emailAddress}
+          id={person.id}
+          isExternal={person.isExternal}
+          isPending={person.isPending}
         />
-      );
-    });
+      </div>
+    );
+  }
+
+  function onRowsRendered({startIndex, stopIndex}) {
+    // gets users from displayed rows that are not labels
+    const userIds = totalRows.slice(startIndex, stopIndex).filter((row) => row.type === 'person').map((p) => p.details.id);
+    onDisplayUsers(userIds);
   }
 
   return (
-    <div>
+    <div style={{height: '100%', width: '100%'}}>
       {
-        items.map((group) => (
-          <div className={classNames('ciscospark-people-group', styles.group)} key={group.label || '1'}>
-            {
-              group.label
-              && <h3 className={classNames('ciscospark-people-group-title', styles.title)}>{group.label}</h3>
-            }
-            {group.people && group.people.length && generateGroupList(group.people)}
-          </div>
-        ))
+        <AutoSizer>
+          {({height, width}) => (
+            <List
+              height={height}
+              rowCount={totalRows.length}
+              rowHeight={44}
```

Comment:
Are the header rows the same size as the people cells? If not, we'll have to make this prop a function to test if it is a header or person and return the correct size

Reply:
yeah, I tried this but the issue is the totalRows array is undefined at the point where the row height gets defined. I'll keep looking in to it today.